pancreabble/handle_notification.py

```diff
@@ -0,0 +1,82 @@
+from libpebble2.services.notifications import Notifications
+from pebble import PebbleUse
+import os.path
+import time
+import json
+import string
+
+SMB_SUGGESTED="enact/smb-suggested.json"
```

Comment:
Why is this file a hard-coded constant while `last_notification_file` is a runtime argument? I think this should be provided as an argument as well.

Reply:
fixed